src/schema/vacation-rental.json

```diff
@@ -142,22 +138,22 @@
             "code": "DepositNotFound",
             "category": "client",
             "description": "Given deposit option does not match one of the options we are expecting"
-        }, 
```

Comment:
This points to a need for Editorconfig files.

Reply:
Perhaps, I just couldn't withstand temptation to make linter happy.

Reply:
Good point though, I will let author of these lines know.